model/client4.go

```diff
@@ -4452,6 +4452,18 @@ func (c *Client4) InstallPluginFromUrl(downloadUrl string, force bool) (*Manifes
 	return ManifestFromJson(r.Body), BuildResponse(r)
 }
 
+// InstallMarketplacePlugin will install marketplace plugin
+// WARNING: PLUGINS ARE STILL EXPERIMENTAL. THIS FUNCTION IS SUBJECT TO CHANGE.
+func (c *Client4) InstallMarketplacePlugin(pluginId, version string) (*Manifest, *Response) {
```

Comment:
i wondering if we should use `PluginRequest` struct here instead of `pluginId, version string`? Just a thought. 0/5 on either approach.